cloud/google/machineactuator_test.go

```diff
@@ -266,20 +268,40 @@ func newMachine(t *testing.T, gceProviderConfig gceconfigv1.GCEProviderConfig) *
 	}
 }
 
-func newGCEProviderConfigFixture() gceconfigv1.GCEProviderConfig {
-	return gceconfigv1.GCEProviderConfig{
+func newGCEMachineProviderConfigFixture() gceconfigv1.GCEMachineProviderConfig {
+	return gceconfigv1.GCEMachineProviderConfig{
 		TypeMeta: v1.TypeMeta{
 			APIVersion: "gceproviderconfig/v1alpha1",
-			Kind:       "GCEProviderConfig",
+			Kind:       "GCEMachineProviderConfig",
 		},
-		Project: "project-name-2000",
 		Zone:    "us-west5-f",
 		OS:      "os-name",
 		Disks:   make([]gceconfigv1.Disk, 0),
 	}
 }
 
-func newDefaultClusterFixture() *v1alpha1.Cluster {
+func newGCEClusterProviderConfigFixture() gceconfigv1.GCEClusterProviderConfig {
+	return gceconfigv1.GCEClusterProviderConfig{
+		TypeMeta: v1.TypeMeta{
+			APIVersion: "gceproviderconfig/v1alpha1",
+			Kind:       "GCEClusterProviderConfig",
+		},
+		Project:    "project-name-2000",
+	}
+}
+
+func newDefaultClusterFixture(t *testing.T) *v1alpha1.Cluster {
+	gceProviderConfig := newGCEClusterProviderConfigFixture()
```

Comment:
Would be a little bit simpler if this line, 294, moved to right after line 298 so that the gceProviderConfig is declared closest to where it is used.